src/Components/HomePage.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import Card from "./RecipeCard";
+import Meal from "../assets/Meal.jpg";
+
+const HomePage = () => {
+  return (
+    <div className="flex flex-col bg-slate-200 p-10 gap-10 overflow-x-hidden w-full">
+      <input
+        type="search"
+        className="h-8 rounded-lg p-4 max-w-full"
+        placeholder="What do you want to cook today?"
+      />
+      <div>
+        <h1>Recently Added</h1>
+        <div className="overflow-x-auto max-w-full">
+          <div className="inline-flex gap-5 w-max">
+            <Card
```

Comment:
Reminder for future once connected to database to map through and render Cards based on how much data is on the database.